src/ui/public/filter_editor/filter_editor.html

```diff
@@ -0,0 +1,132 @@
+<div class="filterEditor kuiModal">
+  <div class="kuiModalHeader">
+    <div class="kuiModalHeader__title">
+      <span ng-show="filterEditor.filter.meta.isNew">Add</span>
+      <span ng-hide="filterEditor.filter.meta.isNew">Edit</span>
+      filter
+    </div>
+
+    <div
+      class="kuiModalHeaderCloseButton kuiIcon fa-times"
+      ng-click="filterEditor.onCancel()"
+    ></div>
+  </div>
+
+  <div class="kuiModalBody">
+    <div ng-show="filterEditor.showQueryDslEditor()">
+      <filter-query-dsl-editor
+        filter="filterEditor.filter"
+        on-change="filterEditor.setQueryDsl(queryDsl)"
+      ></filter-query-dsl-editor>
+      <div
+        ng-show="filterEditor.isEditingQueryDsl"
+        ng-click="filterEditor.isEditingQueryDsl = false"
+      >
+        <button class="kuiMenuButton kuiMenuButton--basic">
+          <span class="kuiMenuButton__icon kuiIcon fa-chevron-left"></span>
+          <span>Back</span>
+        </button>
+      </div>
+    </div>
+
+    <div
+      class="kuiFieldGroup kuiVerticalRhythm"
+      ng-hide="filterEditor.showQueryDslEditor()"
+    >
+      <div class="kuiFieldGroupSection kuiColumn kuiColumn--4">
+        <filter-field-select
+          index-patterns="filterEditor.indexPatterns"
+          field="filterEditor.field"
+          on-select="filterEditor.onFieldSelect(field)"
+        ></filter-field-select>
+      </div>
+
+      <div class="kuiFieldGroupSection kuiColumn kuiColumn--4">
+        <filter-operator-select
+          ng-if="filterEditor.field"
+          field="filterEditor.field"
+          operator="filterEditor.operator"
+          on-select="filterEditor.onOperatorSelect(operator)"
+        ></filter-operator-select>
+      </div>
+
+      <div class="kuiFieldGroupSection kuiFieldGroupSection--wide kuiColumn kuiColumn--4">
```

Comment:
Can we remove the kuiColumn and kuiColumn--4 classes from all of these kuiFieldGroupSection elements? And then create a new class called `filterEditor__wideField` and apply it to this last section?

The visual result will be that this last section will get a little extra room, and the others won't take up more than they need to:

![image](https://cloud.githubusercontent.com/assets/1238659/26739767/fcb6a314-4787-11e7-912b-0b47d9db91d7.png)

The new CSS would be:

```less
.filterEditor__wideField {
  overflow: hidden;
}
```